clients/privacy-center/app/server-utils/loadEnvironmentVariables.ts

```diff
@@ -85,6 +86,10 @@ const loadEnvironmentVariables = () => {
     FIDES_CLEAR_COOKIE: process.env.FIDES_PRIVACY_CENTER__FIDES_CLEAR_COOKIE
       ? process.env.FIDES_PRIVACY_CENTER__FIDES_CLEAR_COOKIE === "true"
       : false,
+    FIDES_KNOWN_PREFERENCE:
+      (process.env.FIDES_PRIVACY_CENTER__FIDES_KNOWN_PREFERENCE as
```

Comment:
when we coerce this, is it case-sensitive at all?

Reply:
not here necessarily, but where this gets consumed does check that. This coercion was to help those instances be happy about that check.

Reply:
Should it not be case sensitive? 🤔

Reply:
just curious! This looks fine 👍